h2o-algos/src/main/java/hex/glrm/GLRM.java

```diff
@@ -1188,6 +1282,33 @@ private Chunk chk_xnew(Chunk[] chks, int c) {
       }
     }
 
+
+    /* same as ArrayUtils.multVecArr() but faster I hope. */
```

Comment:
This deserves a better description. The next person who's going read your code will have no idea what `j` is doing here...